dev-util/astyle/astyle-3.4.9-r1.ebuild

```diff
@@ -19,12 +20,16 @@ SLOT="0/3.2"
 KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~riscv ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
 IUSE="examples java static-libs"
 
-DEPEND="
+BDEPEND="verify-sig? ( sec-keys/openpgp-keys-andresimon )"
```

Comment:
BDEPEND should be last, I'll move it below RDEPEND if you are ok with this

Reply:
Please sort as you like. I prefer alphabetical.